.travis.yml

```diff
@@ -3,22 +3,22 @@ before_install:
   - python --version
 install:
   - pip install .[test]
-script: python -m pytest ./testing/ --cov=idelib-archive
+script: python -m pytest ./testing/ --cov=idelib
 after_success:
-  - codecov
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Is this safe?

Reply:
I'm not a fan but it's how the documentation recommends running it.  At the least, travis should be effectively sandboxed and on public repos it should be fine.
https://github.com/codecov/example-python